mm/Kconfig

```diff
@@ -257,4 +257,8 @@ config MM_PANIC_ON_FAILURE
 	default n
 	depends on DEBUG_MM
 
+config MM_MAP
```

Comment:
the basic map code need always exist, otherwise many drivers willn't work

Reply:
This is a trade off between granularity and simplicity. For now there are just a few drivers which can make use of memory maps & unmap feature.

But always enabling this increases code size for all targets, I don't want to do it at this point.

Having a separate config for it only means that features which use that select it.

Reply:
if so, we need select this config for the driver which support mmap command. But if nobody calls mmap and munmap, the linker will remove all related code from final image. On the other hand, the driver which implement mmap function can't remove even nobody call mmap and munmap since it's very hard to let compiler/linker to remove the reference by pointer.

Reply:
done